libs/portals/admin/petition/src/screens/PetitionList/PetitionList.loader.ts

```diff
@@ -0,0 +1,57 @@
+import type { WrappedLoaderFn } from '@island.is/portals/core'
+import { EndorsementList } from '../../shared/utils/types'
+import {
+  EndorsementSystemGetSingleEndorsementListDocument,
+  EndorsementSystemGetSingleEndorsementListQuery,
+} from '../../shared/queries/getSinglePetitionList.generated'
+import {
+  EndorsementSystemGetEndorsementsDocument,
+  EndorsementSystemGetEndorsementsQuery,
+} from '../../shared/queries/getEndorsements.generated'
+
+export const petitionListLoader: WrappedLoaderFn = ({ client }) => {
+  return async ({ params }): Promise<EndorsementList> => {
+    if (!params.listId) {
+      throw new Error('not found')
+    }
+    const {
+      data: listData,
+      error: listError,
+    } = await client.query<EndorsementSystemGetSingleEndorsementListQuery>({
+      query: EndorsementSystemGetSingleEndorsementListDocument,
+      fetchPolicy: 'network-only',
+      variables: {
+        input: {
+          listId: params.listId,
+        },
+      },
+    })
+    if (listError) {
+      throw listError
+    }
+    if (!listData) {
+      throw new Error('not found')
```

Comment:
Both errors are `not found`.

If these are logged, is it better to be more specific, like?

```
throw new Error('Listid not provided in parameters')
...
...
throw new Error(`No list data found for ${params.listId}`)
```